tests/oss-fuzz/avif_decode_fuzzer.cc

```diff
@@ -43,18 +43,18 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * Data, size_t Size)
                 avifRGBImageSetDefaults(&rgb, decoder->image);
 
                 for (size_t rgbFormatsIndex = 0; rgbFormatsIndex < rgbFormatsCount; ++rgbFormatsIndex) {
-                    for (size_t chromaUpsamplingIndex = 0; chromaUpsamplingIndex < chromaUpsamplingCount; ++chromaUpsamplingIndex) {
+                    for (size_t upsamplingsIndex = 0; upsamplingsIndex < upsamplingsCount; ++upsamplingsIndex) {
                         for (size_t rgbDepthsIndex = 0; rgbDepthsIndex < rgbDepthsCount; ++rgbDepthsIndex) {
                             // Convert to RGB
                             rgb.format = rgbFormats[rgbFormatsIndex];
-                            rgb.chromaUpsampling = chromaUpsamplings[chromaUpsamplingIndex];
-                            rgb.avoidLibYUV = AVIF_TRUE;
                             rgb.depth = rgbDepths[rgbDepthsIndex];
+                            rgb.chromaUpsampling = upsamplings[upsamplingsIndex];
+                            rgb.avoidLibYUV = AVIF_TRUE;
                             avifRGBImageAllocatePixels(&rgb);
                             avifResult rgbResult = avifImageYUVToRGB(decoder->image, &rgb);
-                            // Since avifImageRGBToYUV() ignores chromaUpsamplings, we only need
-                            // to test avifImageRGBToYUV() with a single chromaUpsamplingIndex.
-                            if ((rgbResult == AVIF_RESULT_OK) && (chromaUpsamplingIndex == 0)) {
+                            // Since avifImageRGBToYUV() ignores rgb.chromaUpsampling, we only need
```

Comment:
Only the change in this line is necessary. All the other changes in this pull request are optional, but the original, shorter names make the code a little less verbose.